impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -446,6 +446,18 @@ std::shared_ptr<GlyphAtlas> TypographerContextSkia::CreateGlyphAtlas(
   }
   atlas_context_skia.UpdateBitmap(bitmap);
 
+  // If the new atlas size is the same size as the previous texture, reuse the
+  // texture and treat this as an updated that replaces all glyphs.
+  if (last_atlas && last_atlas->GetTexture() &&
```

Comment:
Pull out last_atlas->GetTexture() so that we don't need to reference it 4 times?

Reply:
Done